example.js

```diff
@@ -0,0 +1,11 @@
+let myPromise;
+
+myPromise.then( 
+  foo => {
+    console.log(foo);
+  },
+  bar => {
+    console.log(bar);
+  }
```

Comment:
Good job done so far @RaviTejaM9602 💪 
 There's nothing else to talk about
Happy Coding 🥳